modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/Template.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.internal.upgrade;
+
+import java.util.regex.Pattern;
+
+/**
+ * @author Felipe Veloso
+ */
+public class Template {
+
+	public Template(
+		String contextVariable, String pattern, String patternReplacement) {
```

Comment:
@velosofelipe64, why did you decide to use `String` as the type of the variable `patternReplacement` instead of use the type `Pattern`?